risc0/circuit/rv32im/src/execute/executor.rs

```diff
@@ -470,6 +486,7 @@ impl<S: Syscall> Risc0Context for Executor<'_, '_, S> {
 
     fn on_terminate(&mut self, a0: u32, a1: u32) -> Result<()> {
         self.user_cycles += 1;
+        // TODO: We don't want to count user_cycles towards the terminate ecall, right?
```

Comment:
Why not?

Reply:
I was thinking of ecall cycles as something separate from user or paging cycles -- that it was a different category and each cycle would be at most one of these. Sounds like this model isn't what we want, so I'll change this.